contracts/Strategy.sol

```diff
@@ -873,75 +671,20 @@ contract Strategy is BaseStrategy, IERC3156FlashBorrower {
         }
     }
 
-    function getTokenOutPathV3(address _token_in, address _token_out)
-        internal
-        view
-        returns (bytes memory _path)
-    {
-        if (address(want) == weth) {
-            _path = abi.encodePacked(
-                address(aave),
-                aaveToWethSwapFee,
-                address(weth)
-            );
-        } else {
-            _path = abi.encodePacked(
-                address(aave),
-                aaveToWethSwapFee,
-                address(weth),
-                wethToWantSwapFee,
-                address(want)
-            );
-        }
-    }
-
-    function _sellAAVEForWant(uint256 amountIn, uint256 minOut) internal {
+    function _sellRewardForWant(uint256 amountIn, uint256 minOut) internal {
         if (amountIn == 0) {
             return;
         }
-        if (swapRouter == SwapRouter.UniV3) {
-            UNI_V3_ROUTER.exactInput(
-                ISwapRouter.ExactInputParams(
-                    getTokenOutPathV3(address(aave), address(want)),
-                    address(this),
-                    now,
-                    amountIn,
-                    minOut
-                )
-            );
-        } else {
-            IUni router =
-                swapRouter == SwapRouter.UniV2
-                    ? UNI_V2_ROUTER
-                    : SUSHI_V2_ROUTER;
-            router.swapExactTokensForTokens(
-                amountIn,
-                minOut,
-                getTokenOutPathV2(address(aave), address(want)),
-                address(this),
-                now
-            );
-        }
-    }
-
-    function _sellSTKAAVEToAAVE(uint256 amountIn, uint256 minOut) internal {
-        // Swap Rewards in UNIV3
-        // NOTE: Unoptimized, can be frontrun and most importantly this pool is low liquidity
-        UNI_V3_ROUTER.exactInputSingle(
-            ISwapRouter.ExactInputSingleParams(
-                address(stkAave),
-                address(aave),
-                stkAaveToAaveSwapFee,
-                address(this),
-                now,
-                amountIn, // wei
-                minOut,
-                0
-            )
+        router.swapExactTokensForTokens(
+            amountIn,
+            minOut,
+            getTokenOutPathV2(address(geist), address(want)),
+            address(this),
+            now
         );
     }
 
-    function getAaveAssets() internal view returns (address[] memory assets) {
+    function getAssets() internal view returns (address[] memory assets) {
```

Comment:
what happens if either aToken or debtToken are not incentivised? 

will it revert?